site/content/operational_guide/resource_limits.md

```diff
@@ -94,6 +94,34 @@ limits:
   maxOutstandingReadRequests: 0
 ```
 
+### Dynamic configuration
+
+Query limits can be dynamically driven by etcd to adjust limits without redeploying. By updating the `m3db.query.limits` key in etcd, specific limits can be overriden. M3Coordinator exposes an API for updating etcd key/value pairs and so this API can be used for modifying these dynamic overrides. For example,
+
+```
+curl -vvvsSf -X POST 0.0.0.0:7201/api/v1/kvstore -d '{
+  "m3db.query.limits",
+  "value":{
+    "maxRecentlyQueriedSeriesDiskBytesRead": {
+      "limit":0,
+      "lookbackSeconds":15,
+      "forceExceeded":false
+    },
+    "maxRecentlyQueriedSeriesBlocks": {
+      "limit":0,
+      "lookbackSeconds":15,
+      "forceExceeded":false
+    }
+  },
+  "commit":true
+}'
+```
+
+Usage notes:
+- The `commit` flag allows for dry-run API calls to see the old and new limits that would be applied.
```

Comment:
nit: 

Setting the `commit` flag to `false` allows for dry-run API calls to see the old and new limits that would be applied.
```suggestion
- The `commit` flag allows for dry-run API calls to see the old and new limits that would be applied.
```